### ISSUE TITLE
Pytorch 1.8 break models

### ISSUE THREAD

**Issue by LoicGrobol** [bencrabbe/npdependency#35]
Using models originally trained in pytorch 1.7.1 with pytorch 1.8 results in a surprising degradation in performances, especially for models not using BERT (the LAS on GSD-UD 2.7 test with no BERT goes from 88.93% to 81.05%). I suspect the origin of the error is that pytorch 1.8 [fixes](https://github.com/pytorch/pytorch/pull/47184) [a bug that made them ignore the padding index in embeddings layers in some cases](https://github.com/pytorch/pytorch/issues/46585). In particular the FastText lexer was trained with this bug (so it expects a non-zero vector for padding subwords).

This is severe (and might have affected some of our results so far), so I will work on fixing it right now, pinning torch to 1.8.0 (to prevent future breakage) and retrain the models. It underlines the need to pin torch to a strict version and to run non-regression performance tests before updating the requirement (sorry for that).
